.github/workflows/test.yml

```diff
@@ -9,12 +9,13 @@ jobs:
     continue-on-error: ${{ matrix.allow-failure }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-22.04]
```

Comment:
Perhaps we should keep this at `ubuntu-24.04`? That's the current value of `ubuntu-latest`, and there's nothing else that would constrain us to `ubuntu-22.04`.